infra/main.tf

```diff
@@ -1,354 +1,147 @@
-    timeout             = 10
-    healthy_threshold   = 5
-    unhealthy_threshold = 5
-    matcher             = "200-399"
-  }
+resource "aws_key_pair" "deployer" {
+  key_name   = "deployer-key"
+  public_key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOgP6TjSCjZS/VWhixYYevHGdzVN4jmlT5KH9va5CiBs elias.samuli@gmail.com"
 }
 
-# Creating a target group for the load balancer
-resource "aws_lb_target_group" "back-target-group" {
-  name        = "back-target-group"
-  port        = "80"
-  protocol    = "HTTP"
-  target_type = "ip"
-  vpc_id      = aws_default_vpc.default_vpc.id # Referencing the default VPC
+data "aws_ami" "amazon_linux" {
+  most_recent = true
+  owners      = ["099720109477"]
 
-  health_check {
-    path                = "/"
-    interval            = 60
-    timeout             = 10
-    healthy_threshold   = 5
-    unhealthy_threshold = 5
-    matcher             = "200-399"
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd-gp3/ubuntu-noble-24.04-amd64-server-*"]
   }
-}
-
-# Creating a client listener for the load balancer
-resource "aws_lb_listener" "client-listener" {
-  load_balancer_arn = aws_alb.front-lb.arn # Referencing our load balancer
-  port              = "80"
-  protocol          = "HTTP"
-
-  default_action {
-    type             = "forward"
-    target_group_arn = aws_lb_target_group.front-target-group.arn # Referencing our target group
+  filter {
+    name   = "architecture"
+    values = ["x86_64"]
   }
 }
 
-resource "aws_lb_listener" "server-listener" {
-  load_balancer_arn = aws_alb.back-lb.arn
-  port              = "80"
-  protocol          = "HTTP"
+resource "aws_ebs_volume" "posrgres" {
+  availability_zone = module.ec2_instance.availability_zone
+  size              = 10
+  type              = "gp3"
 
-  default_action {
-    type             = "forward"
-    target_group_arn = aws_lb_target_group.back-target-group.arn
-  }
+  tags = merge({ Name : "Stagnum-postgers" }, local.tags)
 }
 
-
-# Creating the service
-resource "aws_ecs_service" "stagnum-front-service" {
-  name            = "${var.app_name}-front-service"
-  cluster         = aws_ecs_cluster.aws-cluster.id         # Referencing our created Cluster
-  task_definition = aws_ecs_task_definition.front-task.arn # Referencing the task our service will spin up
-  launch_type     = "FARGATE"
-  desired_count   = 1 # Setting the number of tasks we want deployed
-
-
-  load_balancer {
-    target_group_arn = aws_lb_target_group.front-target-group.arn # Referencing our target group
-    container_name   = "${var.app_name}-front-container"
-    container_port   = 3000 # Specifying the container port
-  }
-
-  network_configuration {
-    subnets          = ["${aws_default_subnet.default_subnet_a.id}", "${aws_default_subnet.default_subnet_b.id}"]
-    assign_public_ip = true
-    security_groups  = ["${aws_security_group.aws-service_security_group.id}"] # Setting the security group
-  }
+resource "aws_volume_attachment" "this" {
```

Comment:
The postgres data should be stored here so that even if the ec2 instance would be destroyed